homeassistant/const.py

```diff
@@ -171,6 +171,8 @@
 EVENT_LOGBOOK_ENTRY = 'logbook_entry'
 EVENT_THEMES_UPDATED = 'themes_updated'
 EVENT_TIMER_OUT_OF_SYNC = 'timer_out_of_sync'
+EVENT_AUTOMATION_TRIGGER = 'automation_trigger'
+EVENT_SCRIPT_RUN = 'script_run'
```

Comment:
I think that we should rename this to `script_started`. With `run` it's ambiguous if it happens before or after all steps of the script are completed.